robot/impl/local_robot.go

```diff
@@ -67,6 +67,10 @@ type localRobot struct {
 	frameSvc framesystem.Service
 }
 
+func (r *localRobot) Dot() ([]byte, error) {
```

Comment:
I think `Dot` is passable for the ResourceManager method name. But the robot API should be more descriptive. E.g: `ExportResourceGraphAsDot`.

I don't think graphviz and subsequently the dot engine are ubiquitous enough to just call things "Dot" and expect readers to have any intuition on what that means. Particularly when we're threading it up through the API like this.